lib/ansiblelint/rules/IncludeMissingFileRule.py

```diff
@@ -46,7 +46,7 @@ def matchplay(self, file, data):
                 if referenced_file:
                     break
 
-            if referenced_file is None:
+            if referenced_file is None or absolute_directory is None:
```

Comment:
This looks like a band-aid. Maybe we should use a relative dir in this case instead?

Reply:
@jlusiardi ^

Reply:
I've merged this PR but I'm pretty sure that this change https://github.com/ansible/ansible-lint/pull/691/files#diff-3a0d2814ee14f24f804a7c3bd1c7863aR266 needs to be applied to other places with `files.append()` in that method.

Reply:
@webknjaz I can rephrase it if you want: I am ok to revert this rule until it gets better testing. I do not think is fair to ask me to fix it, especially as doing any development on ansible-lint is a PITA due to the lack of editable install.